zun/container/k8s/mapping.py

```diff
@@ -192,6 +192,10 @@ def deployment(container, image, requested_volumes=None):
                 "configMap": {"name": vol_name},
             })
 
+    secrets_spec = []
+    if secrets:
+        secrets_spec = [{"name": name} for name in secrets]
```

Comment:
My understanding of the k8s API is that there can only be one secret. The docs don't really specify this but I can't see how it could work any other way. Probably it's fine to leave this way, just expressing my own confusion :p

Reply:
Interesting, I think this probably is fine for now, but if a project has more than one username on a domain, then it would cause collisions.